hil/api.py

```diff
@@ -1289,7 +1289,7 @@ def show_console(nodename):
     if log is None:
         raise errors.NotFoundError(
             'The console log for %s does not exist.' % nodename)
-    return log
+    return json.dumps(log)
```

Comment:
I don't think casting this to JSON is necessarily the right thing here. Once we move this to obmd we'll want to expose the raw byte stream in the client, so maybe we should just make that tweak now.

Reply:
should we clean up in another PR, since we have lots of places are using `json.dumps`